webknossos/webknossos/skeleton/skeleton.py

```diff
@@ -282,5 +282,8 @@ def write(self, out_path: PathLike) -> None:
         warn_deprecated("Skeleton.write", "skeleton.save")
         self.save(out_path)
 
+    def __eq__(self, other: Any) -> bool:
+        return type(other) is type(self) and self._id == other._id
```

Comment:
this class inherits from group which also defines `__eq__`, right? couldn't the method here be omitted?

Reply:
Yes, I think I tested it before, but it didn't work. I just figured that `@attr.define` adds a standard `__eq__` implementation. This can be avoided when adding `@attr.define(eq=False)`. Thanks for mentioning it, I removed the `__eq__` and the `__hash__` method from the `Skeleton` class.